Resources/doc/cookbook/populate-optimizations.md

```diff
@@ -0,0 +1,16 @@
+Populate optimizations
+======================
+
+Here in this chapter we describe some ways to improve performance of `fos:elastica:populate` command. 
+
+Enqueue Elastica Bundle
+-----------------------
+
+The solution utilize [message queue](https://en.wikipedia.org/wiki/Message_queue) by distributing the work among several consumers.
+You can use different transports such as RabbitMQ, Amqp, Stomp, Filesystem and so.
+The performance gain depends on how much consumers you run. 
+For example 10 consumers may give you 5 to 7 times better performance.
```

Comment:
@The-Don-Himself I tested it here https://github.com/php-enqueue/enqueue-sandbox/pull/1.

I created a blog entity generated x thousands of records. Then run elastica populate cmd with `time`. That's how I benchmark it. 

Would it be enough to provide a link to that PR (with some instruction inside it on how t run it) ?